### DISCUSSION TITLE
TEA-DT Health Assurance Cases

### DISCUSSION THREAD

**Discussion by chrisdburr** [alan-turing-institute/AssurancePlatform#502] [General]
Please add your assurance cases to this conversation by Tuesday 25th June (End of Day). You can upload the `json` file by clicking on the 'Paste, drop, or click to add files' button below the text box.

Please name your file with the name of your organisation, so that other participants know which one to vote for (e.g. `alan-turing-institute.json`).

You can then vote for another project team's assurance case by using this form: https://forms.office.com/Pages/ResponsePage.aspx?id=p_SVQ1XklU-Knx-672OE-dxav9xbraJGpD_BP5Yb3HhUMTdXRkRKUzY2T0lLTEs1NkNMMTRBU0g4Si4u 

All votes need to be cast by Friday 28th June. Please note that any votes submitted after this date or for your own project will not be counted.

Winners will be notified on Monday 1st July.

**Comment by alonsoJASL** [comment]
Marina and José's group  

[Link to assurance case](https://github.com/user-attachments/files/15925705/assurance_case.json)

**Reply by chrisdburr** [reply]
> Marina and José's group
> 
> [Link to assurance case](https://github.com/user-attachments/files/15925705/assurance_case.json)

Thanks, @alonsoJASL. Please could you update the filename to include your affiliation (i.e. `imperial-college-london.json`)?

**Reply by alonsoJASL** [reply]
Apologies, here you go 

[imperial-college-london.json](https://github.com/user-attachments/files/15925936/imperial-college-london.json)

**Comment by msinclair-hf** [comment]
Attached is the HeartFlow assurance case.

[heartflow.json](https://github.com/user-attachments/files/16022012/heartflow.json)

**Comment by MaxxHolmes** [comment]
Oxford Group: 
[TEA-DT-Oxford.json](https://github.com/user-attachments/files/16027350/TEA-DT-Oxford.json)
